docs/architecture/metadata-serving.md

```diff
@@ -25,7 +25,7 @@ Note that not all MCP-s will result in an MCL, because the DataHub serving tier
 
 ### Metadata Index Applier (mae-consumer-job)
 
-[Metadata Change Logs]s are consumed by another Spring job, [mae-consumer-job], which applies the changes to the [graph] and [search index] accordingly.
+[Metadata Change Log]s are consumed by another Spring job, [mae-consumer-job], which applies the changes to the [graph] and [search index] accordingly.
```

Comment:
Looks like we might need to add the link reference to the end of this page as wel :)

Reply:
(As you've done above)

Reply:
yeah, there is already a link to the Metadata Change Log, but  no link to the Metadata Change Logs, so I modified it !